.rubocop.yml

```diff
@@ -1,29 +1,169 @@
+inherit_from:
+  - .rubocop_todo.yml
+
 AllCops:
+  TargetRubyVersion: 2.1
+  TargetRailsVersion: 3.2
   Include:
     - '**/Rakefile'
     - '**/config.ru'
   Exclude:
     - 'db/**/*'
     - 'config/**/*'
     - 'script/**/*'
-    - 'spec/**/*'
     - !ruby/regexp /old_and_unused\.rb$/
 
-Documentation:
+# OFN SETTINGS
+# Cop settings that have been agreed upon by the OFN community
+
+Rails:
+  Enabled: true
+
+Style/Documentation:
+  Enabled: false
+
+Style/StringLiterals:
+  Enabled: false
+
+# TEMPORARY/CONTESTED SETTINGS
+# These are still to be decided upon, but recommended for inclusion by
+# oeoeaio after scrutinising offenses the codebase
+
+# Don't think this is a big issue, mostly picking up RPSEC scope definitions
+# with lamdas and RSpec '.to change{}' blocks
+Lint/AmbiguousBlockAssociation:
+  Enabled: false
+
+# No alternative to this one until we upgrade to Rails 4 and can use #find_by
+Rails/DynamicFindBy:
+  Enabled: false
```

Comment:
According to https://github.com/bbatsov/rails-style-guide#find_by both methods are bad and only the Rails 4 `find_by` is good. I would either stick to the old `find_by_attribute` (disable check) or monkey patch ActiveRecord to define `find_by(col, val)` as `where(col => val).first`.